components/builder-depot/habitat/plan.sh

```diff
@@ -0,0 +1,22 @@
+pkg_name=builder-minio
```

Comment:
In the studiorc I think we actually pre-pend `builder` to the services being run. This service is a totally new one and I don't think it . I thought about naming it builder-depot but that felt like a misnomer because its not actually the depot its _just_ minio and nothing else. We can't bundle this in to builder-api either without converting builder-api to a composite plan and that felt like an unnecessary change.

Reply:
It makes me pretty uncomfortable to have builder-depot component package up a service. Builder-depot will get merged into builder-api at some point.  It's not meant to package new services.  I think we should just create a new component that is just builder-minio, and not overload builder-depot.  That should fulfill your needs and keep things cleanly separated.

Reply:
Sure thing. I'll get that switched over.